### PR TITLE
Improve user account prerequisite wording (SCAP)

### PR DESCRIPTION
**Pull request by Lennonka**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

**Comment by Lennonka** [comment]
Please, double check that I didn't miss any occurence. I checked after myself, but two pairs of eyes is better!

File: guides/common/modules/proc_creating-a-compliance-policy.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@ Use this procedure to create a compliance policy, which you can use to ensure th
 * You have available SCAP contents, and eventually tailoring files, in {Project}.
 ** To verify what SCAP contents are available, see xref:listing-available-scap-contents_{context}[].
 ** To upload SCAP contents and tailoring files, see xref:Configuring_SCAP_Contents_{context}[].
-* Your user account has the `view_policies` and `create_policies` permissions.
+* Your user account has a role assigned that has the `view_policies` and `create_policies` permissions.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_creating-a-compliance-policy.adoc:11]: alternative: "a role assigned with X any Y persmissions"?

**Review by maximiliankolb — APPROVED** [review]
LGTM.

File: guides/common/modules/proc_creating-a-compliance-policy.adoc (hunk 1 above)
Comment (Lennonka) [reply guides/common/modules/proc_creating-a-compliance-policy.adoc:11]: That sounds a bit odd, sorry, going with "that has" :D

**Comment by Lennonka** [comment]
Cherry-picked:
- f27eb86e1..4a1323f25  3.7 -> 3.7
- 44b40d5e5..b5cb419d3  3.6 -> 3.6

**Pull request merged**